src/_pytest/stepwise.py

```diff
@@ -8,53 +9,70 @@
 from _pytest.main import Session
 from _pytest.reports import TestReport
 
+if TYPE_CHECKING:
+    from _pytest.cacheprovider import Cache
+
+STEPWISE_CACHE_DIR = "cache/stepwise"
+NO_PREVIOUSLY_FAILED_WONT_SKIP = "no previously failed tests, not skipping."
+PREVIOUSLY_FAILED_TEST_NOT_FOUND = "previously failed test not found, not skipping."
+SKIPPING_PASSED_ITEMS = "skipping {} already passed items."
+
 
 def pytest_addoption(parser: Parser) -> None:
     group = parser.getgroup("general")
     group.addoption(
         "--sw",
         "--stepwise",
         action="store_true",
+        default=False,
```

Comment:
I believe `store_true` implies that the default is false, so it's not really needed. But I suppose it doesn't hurt (assuming it doesn't change the output?)

Reply:
I was kind of confused on the code base regarding this, as its scattered throughout some declaring defaults for store_true and some not, would be good to have a standard going forward, I thought i'd opt for being explicit but either is good :)

Reply:
I personally also like it to be explicit. 👍